ert_gui/model/snapshot.py

```diff
@@ -249,11 +248,16 @@ def _job_data(self, index: QModelIndex, node: Node, role: int):
                     return byte_with_unit(bytes)
             if data_name in [ids.STDOUT, ids.STDERR]:
                 return "OPEN" if node.data.get(data_name) else QVariant()
-            if data_name in [ids.START_TIME, ids.END_TIME]:
-                _time = node.data.get(data_name)
-                if _time is not None:
-                    return str(_time)
-                return QVariant()
+            if data_name in [ids.START_TIME]:
+                start_time = node.data.get(ids.START_TIME)
+                if start_time is None:
+                    return QVariant()
+                end_time = node.data.get(ids.END_TIME) or datetime.datetime.now(
+                    datetime.timezone.utc
```

Comment:
Possible to use the same tz as the time stamps themselves?

Reply:
Yes, absolutely. Otherwise this could have poor results other places in the world.. Date and time for the win I suppose!

Reply:
Done